generators/app/index.js

```diff
@@ -3,20 +3,21 @@
  *--------------------------------------------------------*/
 'use strict';
 
-let Generator = require('yeoman-generator');
-let yosay = require('yosay');
-
-let path = require('path');
-let validator = require('./validator');
-let snippetConverter = require('./snippetConverter');
```

Comment:
please don't make change on current code. the code is coming from vs code and it'll be hard to merge